Mono.Cecil.Cil/MethodBody.cs

```diff
@@ -306,12 +306,8 @@ void RemoveSequencePoint (Instruction instruction)
 			}
 		}
 
-		void UpdateDebugInformation (Instruction removedInstruction, Instruction existingInstruction)
+		void UpdateDebugInformation (Instruction removed_instruction, Instruction existing_instruction)
```

Comment:
Why to change the parameter names, seems the file uses normal C# parameters casing style?